File: Muneef khan 43 (A-1)/JAVA/Q2.java
import java.util.*; 
class Q2 
{ 
    public static void main(String args[]) 
    { 
        Scanner sc= new Scanner(System.in); 
 
        System.out.println("Press 1. for Term Deposit\nPress 2. for Recurring Deposit"); 
        int choice=sc.nextInt(); 
        switch (choice) { 
            case 1: 
                { 
                    double p,r,t; 
                    System.out.print("Enter principal: "); 
                    p=sc.nextInt(); 
                    System.out.print("Enter rate of interest: "); 
                    r=sc.nextInt(); 
                    System.out.print("Enter time period in years: "); 
                    t=sc.nextInt(); 
                    double a=p*Math.pow((1+r/100),t); 
                    System.out.println("Receivable amount: "+a); 
                    break; 
                } 
            case 2: 
                { 
                    double p,r,t; 
                    System.out.print("Enter monthly installment: "); 
                    p=sc.nextInt(); 
                    System.out.print("Enter rate of interest: "); 
                    r=sc.nextInt(); 
                    System.out.print("Enter time period in months: "); 
                    t=sc.nextInt(); 
                    double a=p*t+p * t*(t+1)/2 * r/100 * 1/12; 
                    System.out.println("Receivable amount: "+a); 
                    break; 
                } 
            default: 
                System.out.println("Invalid input press 1 or 2"); 
                break; 
        } 
         
        System.out.println("*************************************************************"); 
        System.out.println("Program Prepared & Executed by:Muneef Khan,  CLass Roll no.: 43"); 
        System.out.println("*************************************************************"); 
        sc.close(); 
    } 
}

File: Muneef khan 43 (A-1)/C++/Q2.cpp
//Q-2) Write a program in C++ to display the sum of the series [ 9 + 99 + 999 + 9999 ...]

#include<bits/stdc++.h>
using namespace std;
int power(int p)
{
    int c=1;
    for (int i = 0; i < p; i++)
    {
       c=10*c; 
    }
    return c;
}
int main()
{
    int n,x=0;
    cout<<"Input Number of terms: ";
    cin>>n;
    int a[n]={0},sum=0;
    for (int i = 0; i < n; i++)
    {
        x=x+(9*power(i));
        cout<<x<<" ";
        a[i]=x;
    }
    for (int i = 0; i < n; i++)
    {
        sum=sum+a[i];
    }
    cout<<endl<<"The sum of the series is: "<<sum;

    cout<<endl<<"*************************************************************"<<endl;
    cout<<"Program Prepared & Executes by:Muneef Khan, Class Roll no.: 43"<<endl;
    cout<<"*************************************************************"<<endl;

    return 0;
}